### PR TITLE
[READY] Adds Guardian Customization

### PR DESCRIPTION
**Pull request by Fikou**
## About The Pull Request

When you get to be a stando you get to customize your color and name
https://www.youtube.com/watch?v=sL_QAJg_MFk
rethemes miner stands to hivelords from the asteroid
to do:

- [x] sanitize

- [x] make less shitcode maybe

## Why It's Good For The Game

makes the experience more customizable and fun

## Changelog
:cl:
add: You can now choose your name and color as a holoparasite/guardian/holocarp!
add: Miner holoparasites are now hivelords!
fix: fixes lings being able to get mechanical holoparasites
/:cl:

### CODE REVIEW

**Comment by Shadowflame909** [comment]
Nothing can deflect the emerald copyright

**Comment by RaveRadbury** [comment]
Ah, the birth of the static stand main.

File: code/modules/mob/living/simple_animal/guardian/guardian.dm [hunk 1]
```diff
@@ -126,10 +116,24 @@ GLOBAL_LIST_EMPTY(parasites) //all currently existing/living guardians
 	if(!summoner)
 		to_chat(src, "<span class='holoparasite bold'>For some reason, somehow, you have no summoner. Please report this bug immediately.</span>")
 		return
-	to_chat(src, "<span class='holoparasite'>You are <font color=\"[namedatum.colour]\"><b>[real_name]</b></font>, bound to serve [summoner.real_name].</span>")
+	to_chat(src, "<span class='holoparasite'>You are a <b<[real_name]</b>, bound to serve [summoner.real_name].</span>")
```
Comment (bobbahbrown) [inline code/modules/mob/living/simple_animal/guardian/guardian.dm]: you are a John

I don't think 'a' should have been added here unless there's something I'm missing

Comment (Fikou) [reply code/modules/mob/living/simple_animal/guardian/guardian.dm]: this only appears when first spawned, before you can choose your name

**Comment by Tlaltecuhtli** [comment]
does it accept unicode and emojis

**Comment by tmtmtl30** [comment]
not 100% sure about this since it makes recognizing stands by their color no longer a factor, but still neat i suppose

**Comment by cacogen** [comment]
do you like fish sticks

**Comment by Fikou** [comment]
>it makes recognizing stands by their color no longer a factor

what did he mean by this

**Pull request merged**